process/scToProtocol/stakingToPeer.go

```diff
@@ -341,6 +342,7 @@ func (stp *stakingToPeer) updatePeerState(
 		if account.GetTempRating() < stp.unJailRating {
 			log.Debug("node is unJailed, setting temp rating to start rating", "blsKey", blsPubKey)
 			account.SetTempRating(stp.unJailRating)
+			account.SetConsecutiveProposerMisses(0)
```

Comment:
flag ?

Reply:
not yet checked with importdb to see if we need the activation flag